### PR TITLE
improve description for Swift Package Manager

### PR DESCRIPTION
**Pull request by sahara-ooga**
Change description for Swift Package Manager in README.md.
Thanks for review!

I refer to README.md in [apple/swift-argument-parser](https://github.com/apple/swift-argument-parser) in writing it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,11 +293,31 @@ github "YusukeHosonuma/SwiftPrettyPrint"
...
+        .package(url: "https://github.com/YusukeHosonuma/SwiftPrettyPrint.git", .upToNextMajor(from: "1.0.0")),
+        // other dependencies
+    ],
+    targets: [
+        .target(name: "<your-target-name>", dependencies: [
+            .product(name: "SwiftPrettyPrint", package: "SwiftPrettyPrint"),
+        ]),
+        // other targets
+    ]
+)
```
Comment (sahara-ooga) [inline README.md]: This target description is needed in fact.

File: README.md [hunk 2]
```diff
@@ -293,11 +293,31 @@ github "YusukeHosonuma/SwiftPrettyPrint"
...
 ### Swift Package Manager
 
+Add the following line to the dependencies in your `Package.swift` file:
+
 ```swift
 .package(url: "https://github.com/YusukeHosonuma/SwiftPrettyPrint.git", .upToNextMajor(from: "1.0.0"))
 ```
 
-or add from Xcode 10+.
+Finally, include "SwiftPrettyPrint" as a dependency for your executable target:
```
Comment (YusukeHosonuma) [inline README.md]: This library is not locked in to executable target.

```suggestion
Finally, include "SwiftPrettyPrint" as a dependency for your any target:
```

File: README.md [hunk 3]
```diff
@@ -293,11 +293,31 @@ github "YusukeHosonuma/SwiftPrettyPrint"
...
+let package = Package(
+    // name, platforms, products, etc.
+    dependencies: [
+        .package(url: "https://github.com/YusukeHosonuma/SwiftPrettyPrint.git", .upToNextMajor(from: "1.0.0")),
+        // other dependencies
+    ],
+    targets: [
+        .target(name: "<your-target-name>", dependencies: [
+            .product(name: "SwiftPrettyPrint", package: "SwiftPrettyPrint"),
+        ]),
```
Comment (YusukeHosonuma) [inline README.md]: The `dependencies` can described by name when package name and library name are same.

```suggestion
        .target(name: "<your-target-name>", dependencies: ["SwiftPrettyPrint"]),
```

**Review by YusukeHosonuma — COMMENTED** [review]
Good improvements, please confirm to comment.

**Comment by sahara-ooga** [comment]
I imported suggestions ! I appriciate your review & comments!

**Pull request merged**